prep-html-links/contact-us/contact-us.html

```diff
@@ -7,6 +7,11 @@
   </head>
   <body>
     <h1>Contact Us</h1>
-    <div><a href="index.html">Click here to go to the home page</a></div>
+    <div>
+      <a
+        href="file:///C:/Users/sgukasyan/repos/prep-1121-code-solutions/prep-html-links/index.html"
+        >Click here to go to the home page</a
+      >
```

Comment:
Try to use a relative URL for your `href`. Otherwise, your code won't work on anyone else's computer, since this URL has your username in it. For example:

```html
<a href="../index.html">Click here to go to the home page</a>
```